src/components/ProfilePage.js

```diff
@@ -94,12 +73,11 @@ export default function ProfilePage() {
                 firstName: firstName,
                 lastName : lastName,
                 email : email,
-                gender : gender,
```

Comment:
I see in multiple places that you are not including the gender of the user, are we not incorporating that?